holidays/countries/guyana.py

```
@@ -0,0 +1,278 @@
+        self._add_observed(self._add_diwali_india(tr("Deepavali")))
+
+        # Prophet's Birthday.
+        for dt in self._add_mawlid_day(tr("Youman Nabi")):
+            self._add_observed(dt)
+
+        # Eid al-Adha.
+        for dt in self._add_eid_al_adha_day(tr("Eid-Ul-Azha")):
+            self._add_observed(dt)
+
+
+class GY(Guyana):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)